src/components/AttachmentModal.js

```diff
@@ -431,6 +463,16 @@ function AttachmentModal(props) {
                 confirmText={translate('common.close')}
                 shouldShowCancelButton={false}
             />
+            <ConfirmModal
```

Comment:
I added this as a separate modal because it feels like a completely different thing from the one above, but we could probably "fuse" them with a bunch of conditionals

Reply:
I agree this is more clean, but i wonder if we should just completely not render this one or the other one depending on the prop `isAttachmentReceipt`?